### PR TITLE
Fix7320 font image source don't work on wpf

### LINKED ISSUE #7320: [Bug] FontImageSource don't work on wpf
### Description
Using a FontImageSource on a project that is started with a WPF frontend the image or imagebutton control don't display the symbol
### Steps to Reproduce
Put an imagebutton control in a page and use a fontimagesource as the source of the image
### Expected Behavior
The symbol is displayed
### Actual Behavior
The symbol is not displayed
### Basic Information

- Version with issue:
4.2.0.709249
- Last known good version:
- IDE:
Visual studio 2019
- Platform Target Frameworks: 
WPF
- Android Support Library Version: 
NA
- Nuget Packages:
Xamarin.Forms
Xamarin.Forms.Platform.WPF
- Affected Devices:
Windows 10 PC 

### Reproduction Link
[Test.zip](https://github.com/xamarin/Xamarin.Forms/files/3557986/Test.zip)

### PR DESCRIPTION
**Pull request by yurkinh**
### Description of Change ###

Using a FontImageSource on a project that is started with a WPF frontend the image or imagebutton control don't display the symbol

### Issues Resolved ### 
- fixes #7320 

### API Changes ###
None 

### Platforms Affected ### 
- WPF

### Behavioral/Visual Changes ###
Symbol is displayed in Button and ImageButton

### CODE REVIEW

File: Xamarin.Forms.Platform.WPF/Renderers/ImageRenderer.cs [hunk 1]
```diff
@@ -159,4 +159,72 @@ public Task<System.Windows.Media.ImageSource> LoadImageAsync(ImageSource imageso
...
+					FontWeights.Normal,
+					FontStretches.Normal,
+					fontsource.Size,
+					CreateBrush(fontsource.Color));
+			return Task.FromResult(image);
+		}
+
+		private static Brush CreateBrush(Color color)
+		{
+			var wpColor = System.Windows.Media.Color.FromArgb(
```
Comment (campersau) [inline Xamarin.Forms.Platform.WPF/Renderers/ImageRenderer.cs]: You can use the extension method from here:
https://github.com/xamarin/Xamarin.Forms/blob/69171055d6b218d24747cfe5b7beb200344f5317/Xamarin.Forms.Platform.WPF/Extensions/ColorExtensions.cs#L17-L20

Comment (yurkinh) [reply Xamarin.Forms.Platform.WPF/Renderers/ImageRenderer.cs]: Thanks @campersau . I have updated  PR with your suggestion

**Comment by VladislavAntonyuk** [comment]
I had the same issue with macOS and GTK. Could you please take a look to completely fix the issue with font image source on all platforms?

**Comment by yurkinh** [comment]
> I had the same issue with macOS and GTK. Could you please take a look to completely fix the issue with font image source on all platforms?

Actually it's not an issue . FontImageSource handlers are not implemented for mentioned platforms. I think that for macOS we can probably reuse one from iOS with minor modifications but for GTK it should be implemented from the scratch.

File: Xamarin.Forms.Platform.WPF/Renderers/ImageRenderer.cs [hunk 2]
```diff
@@ -159,4 +159,62 @@ public Task<System.Windows.Media.ImageSource> LoadImageAsync(ImageSource imageso
...
+		private static System.Windows.Media.ImageSource CreateGlyph(
+			string text,
+			FontFamily fontFamily,
+			FontStyle fontStyle,
+			FontWeight fontWeight,
+			FontStretch fontStretch,
+			double fontSize,
+			Brush foreBrush)
+		{
+			if (fontFamily != null && !string.IsNullOrEmpty(text))
```
[the remark this answers is not in the document]
Comment (yurkinh) [reply Xamarin.Forms.Platform.WPF/Renderers/ImageRenderer.cs]: Done

File: Xamarin.Forms.Platform.WPF/Renderers/ImageRenderer.cs [hunk 3]
```diff
@@ -159,4 +159,62 @@ public Task<System.Windows.Media.ImageSource> LoadImageAsync(ImageSource imageso
...
+					new FontFamily(new Uri("pack://application:,,,"), fontsource.FontFamily),
+					FontStyles.Normal,
+					FontWeights.Normal,
+					FontStretches.Normal,
+					fontsource.Size,
+					fontsource.Color.ToBrush());
+			return Task.FromResult(image);
+		}		
+
+		private static System.Windows.Media.ImageSource CreateGlyph(
```
[the remark this answers is not in the document]
Comment (yurkinh) [reply Xamarin.Forms.Platform.WPF/Renderers/ImageRenderer.cs]: Done

**Comment by yurkinh** [comment]
@rmarinho  I tested this handler on sample app provided with bug description. 
I will check it with gallery sample

**Comment by yurkinh** [comment]
@rmarinho Finally fixed. 
1. There was incorrect path for fonts folder in WPF and font was missing
2. There was incorrect default color conversion that makes all font images transparent

pic

**Pull request merged**